couler/tests/volume_test.py

```diff
@@ -0,0 +1,17 @@
+import os
```

Comment:
This looks like an integration test so perhaps move it to integration tests folder and make sure to test this on CI as well similar to others.